docs/release/release_0_3_0.md

```diff
@@ -0,0 +1,265 @@
+where the core team will devote effort in the coming months.
+
+We are still overwhelmed by the community response to napari and we hope that
+the above documents will continue to encourage potential users to join our
+community. We continue to welcome contributions of all kinds and encourage
+you to get in touch with us if you don't see your most wanted feature in our
+roadmap, or as an issue in our [issue
+tracker](https://github.com/napari/napari/issues).
+
+### Getting in touch
+
+We have also created a [Zulip chat room](https://napari.zulipchat.com) and
+worked with the [image.sc forum](https://image.sc) (use the "napari" tag) to
+allow users to get help with any issues they might have using napari.
+
+### IO plugins
+
+napari is now extensible to open and save to a wide variety of file formats,
+both local and remote. Currently, the same file formats as before are available
+to read (TIFF, most image file formats supported by imageio, and zarr).
+However, we can now *write* to all these formats, read and write point
+annotations in .csv format, and we also have the ability for anyone to create
+packages for napari to read and write in any other formats. You can read about
+our plugin architecture [here](and https://napari.org/docs/plugins/index.html).
+
+Want to drag and drop your favorite file format into napari? See [this
+guide](https://napari.org/docs/plugins/for_plugin_developers.html) to
+understand how to write your own plugin, and see Jackson Maxfield's
+[napari-aicsimageio](https://github.com/AllenCellModeling/napari-aicsimageio)
+plugin for an example plugin!
+
+Many thanks to Talley Lambert for driving this effort!
+
+### Dockable widgets and magicgui
+
+Another brainchild of Talley is our dockable widget architecture, which allows
+you to pop out the napari UI elements from the main window, enabling, for
+example, those on multi-monitor setups to have the toolbars on one monitor and
+the main window in full-screen on another.
+
+Even better, we have released a side package called `magicgui` to allow you to
+create your own dockable widgets with which to interact with napari. We are
+still working on standard models of interaction here (see our
+[roadmap](https://napari.org/docs/developers/ROADMAP_0_3.html)), but you should
+be able to get started creating useful user interfaces right now. [This
+image.sc
+post](https://forum.image.sc/t/integration-of-napari-module-subclass-plugin/36018/2)
+by Talley provides a good overview of how to create interaction with napari
+right now, and [this GitHub
+answer](https://github.com/napari/napari/issues/1165#issuecomment-618013894)
+explains how to embed a matplotlib plot within napari.
+
+### Multiscale image handling
+
+napari is now much better at handling large datasets. Viewing a large dataset
+will no longer trigger automatic — but very slow, and often unnecessary —
+generation of an image pyramid. Instead, we direct users to our [tutorial on
+how to generate your own pyramid]().
```

Comment:
This link needs to be added

Reply:
I also need to add a note about the viewport bug fix.

Reply:
added a note